lib/src/HttpFileImpl.cc

```diff
@@ -41,6 +41,17 @@ int HttpFileImpl::save(const std::string &path) const
         fsPath = fsUploadPath / fsPath;
     }
     filesystem::path fsFileName(utils::toNativePath(fileName_));
+    if (!filesystem::exists(fsPath))
+    {
+        LOG_TRACE << "create path:" << fsPath;
+        drogon::error_code err;
+        filesystem::create_directories(fsPath, err);
+        if (err)
+        {
+            LOG_SYSERR;
```

Comment:
Nothing is logged?

Reply:
LOG_SYSERR wil print some info of errorno

Reply:
Ohh. Ok